src/main/java/danta/jahia/util/PropertyUtils.java

```diff
@@ -92,7 +93,13 @@ public static Object distill(Property property)
                     List<String> distilledValues = new ArrayList<>();
                     for (int i=0;i<values.length; i++){
                         String referenceUUID = values[i].getString();
-                        distilledValues.add(ResourceUtils.getResourceNodePath(property.getSession(), referenceUUID));
+                        try {
+                            distilledValues.add(ResourceUtils.getResourceNodePath(
+                                    property.getSession(), referenceUUID));
+                        } catch (ItemNotFoundException e ) {
+                            LOG.error("LayerX Exception: Reference to invalid item (e.g., unpublished content): " +
+                                    e.getMessage(),e);
```

Comment:
Some strange spacing:
*  `ItemNotFoundException e ` (space after e)
*  `e.getMessage(),e)` (no space after comma)

Reference to LayerX instead of Danta.